CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## edge-19.9.3
+
+* Helm
+  * Allow disabling namespace creation during install (thanks @KIVagant!)
+* Controller
+  * Fixed proxy injector timeout during a large number of concurrent injections
+  * Separated the destination controller into its own separate pod
+  * Updated Prometheus config to keep only needed cadvisor metrics
+* Web UI
+  * Fixed bad request in the top routes tab on empty fields (thanks @pierDipi!)
+* Proxy
+  * Fixes to the client's backoff logic
+  * Add 587 to the list of ports to disable protocol detection
```

Comment:
fwiw, this was an external PR too https://github.com/linkerd/linkerd2-proxy/pull/350, if you want to call out the author